doc/guide/Guide_Euler.ipynb

```diff
@@ -274,9 +274,9 @@
  ],
```

Comment:
Just a question to @emanuel-schmid  for the beginning of the tutorial: Is it not recommended to use the `/work/` directory for computations?

Reply:
`/cluster/work` and '/cluster/project` are both valid choices. The differences are:

a) `project` may have snapshots. But I think it doesn't in our case.
b) `work` is a high performance file-system, optimized for parallel IO and large files.

Both have to be bought by share holders. In our case both are available.\
The canonical choice is probably `project`, as I guess high performance in parallel IO is not really an issue.

Reply:
Thanks for the clarification! @zeliest could you maybe add this to the guide: `project/` is the default place, but if people need very high performance (probably very rare), `work/climate/username` is possible too.

Reply:
https://github.com/CLIMADA-project/climada_python/pull/181/commits/342b060b49d893f0aa306b7a222f59bdb065d4a3